singer_runner/state/file.py

```diff
@@ -15,7 +15,9 @@ def dump(self, state):
         self.state = state
 
     def load(self):
-        with smart_open(self.filepath, 'r') as file:
-            state = json.load(self.file)
-        self.state = state
-        return state
+        try:
+            with smart_open(self.filepath, 'r') as file:
```

Comment:
Is this to handle `self.filepath` being `None` or the file not being found. If it's a none check, we could add an if conditional here, if it's a file not found, we should probably raise an error.

Reply:
I personally prefer to not have an error if the file doesn't exist as it saves the user from having to create the file first (which requires them to know the content must be valid JSON -- an empty file doesn't work) and then upload it to the bucket. On subsequent calls for incremental updates, say, I can see how the file not being present could be problematic. Perhaps we could have a parameter than indicates if you want to create the state file if it doesn't exist?

Reply:
Sorry, it handles the case where the file doesn't exist or the file is empty.

Reply:
Ahh ok, I see you're excilitable catching `ValueError` too, which should be fine